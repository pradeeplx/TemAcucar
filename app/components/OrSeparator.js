import React from 'react'
import { fontFactor } from "../helpers"
import Colors from "../Colors"
import TextBox from "./TextBox"

export default OrSeparator = () => (
  <TextBox style={{
    margin: 10,
    fontSize: 14 * fontFactor(),
    color: Colors.gray,
  }}>
    ou
  </TextBox>
)
