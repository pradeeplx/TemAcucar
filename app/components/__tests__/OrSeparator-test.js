import 'react-native'
import React from 'react'
import OrSeparator from '../OrSeparator'
import renderer from 'react-test-renderer'

it('renders correctly', () => {
  const tree = renderer.create(
    <OrSeparator />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
