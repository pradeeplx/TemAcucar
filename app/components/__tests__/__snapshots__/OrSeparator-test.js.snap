exports[`test renders correctly 1`] = `
<Text
  accessible={true}
  allowFontScaling={true}
  ellipsizeMode="tail"
  style={
    Array [
      Object {
        "color": "#654C50",
        "fontFamily": "Avenir",
        "fontSize": 18,
      },
      Array [
        Object {
          "marginHorizontal": 16,
          "textAlign": "center",
        },
        Object {
          "color": "#CECED2",
          "fontSize": 21,
          "margin": 10,
        },
      ],
    ]
  }>
  ou
</Text>
`;
